www/api/class/centreon_monitoring_metric.class.php

```diff
@@ -98,6 +98,21 @@ public function getList()
         }
 
         $query .= ' ORDER BY `metric_name` COLLATE utf8_general_ci ';
+
+        if (isset($this->arguments['page_limit']) && isset($this->arguments['page'])) {
+            if (
+                filter_var(($limit = $this->arguments['page_limit'] ?? false), FILTER_VALIDATE_INT) === false
+                || filter_var(($page = $this->arguments['page'] ?? false), FILTER_VALIDATE_INT) === false
```

Comment:
you've already checked that $arguments['page_limit'] and $arguments['page_limit'] are set
```suggestion
                filter_var(($limit = $this->arguments['page_limit']), FILTER_VALIDATE_INT) === false
                || filter_var(($page = $this->arguments['page']), FILTER_VALIDATE_INT) === false
```